script/Ciel/Day 11/11_01_CIEL11_2_1.txt

```diff
@@ -237,7 +237,7 @@ You vampires can keep your perverse logic to yourselves.
 [sha:4b72095a49dda190485cb27299fa9a267dad47b2]{
 -- Page 110, Offset 29370.
 -- 「ひぃ……ひぃぃ、ひぃいいいい……」
-"Ahh... Ahh――AhhhhHhhH..."
+"Ahh... Ahh――Ahhhhhhhh..."
```

Comment:
I'd vote to keep it FuNkY, or at least have it curve up (AhhhHHHH) assuming the VA is screams